### PR TITLE
[CHORE] flask restplus initial build

### PR DESCRIPTION
**Pull request by snazzer**
Initial checkin to get things built and into our pypi. Note the marker on the version tag so we know it's ours.

### CODE REVIEW

File: .circleci/config.yml~ [hunk 1]
```diff
@@ -0,0 +1,71 @@
+version: 2
```
Comment (ncoish) [inline .circleci/config.yml~]: @snazzer Why are there 2, very similar config files? Is one just the original unmodified?

**Review by ncoish — APPROVED** [review]
Looks good, just a question about why there are two circleci configs.

File: .circleci/config.yml~ (hunk 1 above)
Comment (DroneKid) [reply .circleci/config.yml~]: This looks like vim backup file :)

Comment (snazzer) [reply .circleci/config.yml~]: Fixed - thx.

**Pull request merged**